### PR TITLE
Add unit tests

### LINKED ISSUE #15: add more unit tests
I will be adding more unit tests to improve code coverage

### PR DESCRIPTION
**Pull request by avelynhc**
fixes #15 
- added unit tests for method, `process_text_file` and `contains_italics`
- change from `exit()` to `raise` when handling file not found error in `txt2html.py`
- change `.` location before proceed to add more unit tests in `tests/test_process_line.py`

result
Screenshot 2023-11-14 at 1 44 54 PM

### CODE REVIEW

**Comment by mingming-ma** [comment]
@avelynhc Hi, thanks a lot for your contribution, it seems you use another framework which is supported by Pytest, if you have time can you convert them to Pytest style? 🙏

**Comment by avelynhc** [comment]
@mingming-ma is this better? let me know. Thanks!

Screenshot 2023-11-15 at 7 31 45 PM

File: tests/test_process_line.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+from txt2html import process_line, process_text_file, contains_italics
+from unittest.mock import patch
 
 
 @pytest.mark.parametrize(
     "input_line, expected_output",
     [
         ("This is a normal line.", "This is a normal line."),
-        ("This should be *italic*.", "This should be <i>italic</i>."),
+        ("This should be *italic.*", "This should be <i>italic.</i>"),
```
Comment (mingming-ma) [inline tests/test_process_line.py:10]: Bug fixed, no needed here

File: tests/test_process_line.py [hunk 2]
```diff
@@ -18,3 +19,55 @@
 def test_process_line(input_line, expected_output):
     result = process_line(input_line)
     assert result == expected_output
+
+    # test the .txt file
+    @patch("builtins.print")
```
Comment (mingming-ma) [inline tests/test_process_line.py:24]: seems indent should also update after the change, and following cases also

**Review by mingming-ma — CHANGES_REQUESTED** [review]

**Comment by avelynhc** [comment]
@mingming-ma I just made some changes as per your request. Please let me know if you need anything else. Thanks!

**Comment by mingming-ma** [comment]
@avelynhc Thanks for your PR

**Pull request merged**